opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -508,4 +551,8 @@ message SummaryDataPoint {
   // A list of values at different quantiles of the distribution calculated
   // from the current snapshot. The quantiles must be strictly increasing.
   repeated ValueAtQuantile quantile_values = 6;
+
+  // (Optional) List of exemplars collected from 
+  // measurements that were used to form the data point
+  repeated RawValue exemplars = 7;
```

Comment:
Would this just be a list of random samples from the whole window? [Open question](https://github.com/open-telemetry/oteps/blob/master/text/metrics/0113-exemplars.md#open-questions) in the OTEP:

> We don’t have a strong grasp on how the sketch aggregator works in terms of implementation - so we don’t have enough information to design how exemplars should work properly.

Reply:
The proto does not define how the exemplars were sampled, not sure your question?

Reply:
Nvm, I see now

Reply:
@bogdandrutu does this sound right to you, for now?

Reply:
>We also agreed to move the Exemplars into the DataPoints so that they can refer to dropped labels, not include full label sets in each point.

I think we agreed that you will evaluate what is better for performance/semantics:
1. Having a `repeated RawValue exemplars` in the `Metric` that applies to all data points (user may need to do another remapping to every data point) vs `repeated RawValue exemplars` in every point (if we go with every point then "dropped_labels" is better name for that).

My point is that I don't have a strong opinion between the both, and was trying to make you investigate and decide which way. Here are my thoughts:
* Having `repeated RawValue exemplars` in the Metrics:
  * Pros:
    * Saves some memory in the internal representation (have extra 24 bytes per data point).
    * Same message may be able to be re-used with raw-measurements because labels don't represent dropped.
  * Cons:
     * Duplicate labels on the wire.
     * User needs to re-map every exemplar to the data point by doing the labels matching.

I feel `cons` are more "significant" than `pros`, so personally I would go with exemplars in every DataPoint as you suggested.

Reply:
If we go with exemplars in every DataPoint I would say to rename the message to "Exemplar" :)

Reply:
This discussion makes me want a way to intern label sets to avoid the "re-map every exemplar" problem.

I don't feel inclined to invest time in this now, so we should probably choose "repeated RawValue exemplars in every point".

Reply:
> This discussion makes me want a way to intern label sets to avoid the "re-map every exemplar" problem.

Even with an "intern label" you still need to map every exemplar to a point (that mapping may be faster if we have "intern label" but still needs some work)